cobra/test/test_io_order.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from operator import attrgetter
+from os.path import join, dirname
+from random import sample
+
+import pytest
+
+import cobra.io as cio
+from cobra import DictList
+
+
+@pytest.fixture(scope="module")
+def tmp_path(tmpdir_factory):
+    return str(tmpdir_factory.mktemp("model_order"))
+
+
+@pytest.fixture(scope="module")
+def minimized_core():
+    model = cio.read_sbml_model(join(dirname(__file__), "data",
+                                "textbook.xml.gz"))
+    model.id = "minimized_core"
+    chosen = sample(model.reactions, 10)
+    model.remove_reactions(set(model.reactions).difference(chosen),
+                           remove_orphans=True)
```

Comment:
Don't you risk removing all the metabolites if the 10 chosen reactions don't overlap?

Reply:
The metabolites of those reactions should remain, right? That seems like enough metabolites to play with.

Reply:
Oh I guess you're right, nevermind then, yeah that's probably enough